.travis.yml

```diff
@@ -3,8 +3,8 @@ dist: focal
 
 language: go
 go:
+- 1.16.x
 - 1.15.x
-- 1.14.x
```

Comment:
If you remove 1.14, please check line 32 where we run code coverage for 1.14 (and race tests for 1.15). Adjust to the new versions.

Reply:
While we are at it, if we plan to release as 1.16, then put 1.16 line 41